content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -152,6 +152,30 @@ host's file owner/group.
 
 [CVE-2021-25741]: https://github.com/kubernetes/kubernetes/issues/104980
 
+## Integration into Pod Security Standards
+
+{{< feature-state state="alpha" for_k8s_version="v1.29" >}}
+
+All available [Pod Security Standards (PSS) profiles](/docs/concepts/security/pod-security-standards)
+will relax in a controlled way for pods which enable user namespaces. This
```

Comment:
How about:
```suggestion
For Linux Pods that enable user namespaces, Kubernetes relaxes the application of
[Pod Security Standards](/docs/concepts/security/pod-security-standards) in a controlled way.
This
```

Reply:
Do you wanna change to pod security admission and leave the link to pod security standards here? :thinking: 

I applied this change